private/release/choose-semverness.js

```diff
@@ -4,6 +4,7 @@ import { createWriteStream, mkdirSync, readFileSync } from 'node:fs'
 import { spawnSync } from 'node:child_process'
 
 import prompts from 'prompts'
+import { TARGET_BRANCH } from './config.js'
```

Comment:
This has landed on `main` (5a09823f6d725974a657e8c9e6438fa508d5965b), removing it from this PR so it doesn't get reverted when we decide to release 3.x as stable.
```suggestion
```